.github/workflows/_deploy_kafka_test.yml

```diff
@@ -0,0 +1,198 @@
+name: Setup Workload Identity 
+
+on:
+  workflow_call:
+    inputs:
+      skr-client-akv-endpoint:
+        description: "AKV endpoint for key release"
+        required: true
+        type: string
+      skr-client-maa-endpoint:
+        description: "Attestation endpoint"
+        required: true
+        default: "sharedeus2.eus2.test.attest.azure.net"
+        type: string
+      skr-client-kid:
+        description: "Key release key id"
+        required: true
+        default: "kafka-encryption-demo"
+        type: string
+      skr-kafka-demo-topic:
+        description: "Kafka topic"
+        required: true
+        default: "kafka-demo-topic"
+        type: string
+      cluster-name:
+        required: true
+        description: AKS Cluster Name With Randomly Generated Suffix
+        type: string
+      sidecar-registry:
+        required: true
+        description: "The registry to get the image from"
```

Comment:
Use the defaults for this from the ACI examples repo

Reply:
ok